src/chart/onInit.js

```diff
@@ -25,5 +25,5 @@ export default function onInit() {
     removeInvalidControls.call(this);
 
     //Initialize listing.
-    this.listing.init(this.raw_data);
+    this.listing.init(this.raw_data, this.listing.test);
```

Comment:
whats going on with this guy?

Reply:
webcharts has logic that expects the containing element to have been rendered before it proceeds.  but the headless browser we use for testing doesn't actually render the page so there's some logic in `init` that avoids the rendered logic when running unit tests.

Reply:
gotcha gotcha - makes sense